app/states/texas/texas_downloader.py

```
@@ -7,90 +9,271 @@
-        if overwrite:
-            read_from_temp = False
-            for file_extension in ('*.csv', '*.txt'):
-                files = tmp.glob(file_extension)
-                for file in files:
-                    ic(f"Removing {file}")
-                    file.unlink()
-
-        driver = webdriver.Chrome(options=options)
-        wait = WebDriverWait(driver, 10)
+            cls,
+            read_from_temp: bool = True,
```

Ruff (0.14.7)

Boolean-typed positional argument in function definition

(FBT001)